src/plugins/contributionReward/proposal.ts

```diff
@@ -1,23 +1,16 @@
 import BN from 'bn.js'
 import { DocumentNode } from 'graphql'
 import gql from 'graphql-tag'
-import { from, Observable } from 'rxjs'
-import { concatMap } from 'rxjs/operators'
+import { Observable } from 'rxjs'
 import {
   Address,
   Arc,
   ContributionRewardPlugin,
   IApolloQueryOptions,
   IProposalState,
-  ITransaction,
-  ITransactionReceipt,
   Logger,
-  NULL_ADDRESS,
-  Operation,
   Plugin,
-  Proposal,
-  REDEEMER_CONTRACT_VERSIONS,
-  toIOperationObservable
+  Proposal
```

Comment:
why this is added here ?

Reply:
It wasn't added, just the comma was removed and github shows this as a new line. 
The comma is removed because the next imports are removed.